bears/general/KeywordBear.py

```diff
@@ -2,7 +2,66 @@
 
 from coalib.bearlib import deprecate_settings
 from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
 from coalib.results.Result import RESULT_SEVERITY, Result
+from coalib.results.SourceRange import SourceRange
+
+from bears.general.AnnotationBear import AnnotationBear
+
+
+def _get_comments(dependency_results):
+    if not dependency_results:
+        return
+
+    annotation_bear_results = dependency_results.get('AnnotationBear')
+    if (not annotation_bear_results or
+            not isinstance(annotation_bear_results, list)):
+        return
+
+    for result in annotation_bear_results:
+        yield from result.contents.get('comments', [])
+
+
+def generate_diff(comments, file, filename,
+                  line, line_number, pos):
+    todo_source_range = SourceRange.from_values(filename, line_number,
+                                                pos + 1)
+    affected_comment_sourcerange = [
+        c for c in comments if todo_source_range in c]
+    if len(affected_comment_sourcerange) == 0:
+        return {}
+    elif len(affected_comment_sourcerange) > 1:
```

Comment:
ah you need to put a `# pragma: no cover` command here or just replace the elif by an `assert len..., "More than 1 ..."` which is probably simpler